appf.adoc

```diff
@@ -4,167 +4,122 @@
 [appendix]
 == Grid Mappings
 
-Each recognized grid mapping is described in one of the sections below. Each section contains: the valid name that is used with the **`grid_mapping_name`** attribute; a list of the specific attributes that may be used to assign values to the mapping's parameters; the standard names used to identify the coordinate variables that contain the mapping's independent variables; and references to the mapping's definition or other information that may help in using the mapping. Since the attributes used to set a mapping's parameters may be shared among several mappings, their definitions are contained in a table in the final section. The attributes which describe the
-    ellipsoid and prime meridian may be included, when applicable, with
-    any grid mapping. 
+Each recognized grid mapping is described in one of the sections below.
+Each section contains: the valid name that is used with the
+**`grid_mapping_name`** attribute; a list of the specific attributes
+that may be used to assign values to the mapping's parameters; the
+standard names used to identify the coordinate variables that contain
+the mapping's independent variables; and references to the mapping's
+definition or other information that may help in using the mapping.
+Since the attributes used to set a mapping's parameters may be shared
+among several mappings, their definitions are contained in a table in
+the final section. The attributes which describe the ellipsoid and prime
+meridian may be included, when applicable, with any grid mapping.
 
```

Comment:
Explicitly breaking these lines appears to make no difference to the pd for html output.  Only lines 8 and 9 with their leading space have any formatting effect.
We've not enforced a maximum line length in the content overall.

So, do you think that this line breaking adds readability for the editors?
Is this the intent of the change, or is the change only targeting the type spacing of the PDF output?

(similarly for numerous changes in this PR)